scripts/UploadSignedReleaseArtifacts.ps1

```diff
@@ -33,6 +33,13 @@ $org = 'https://dev.azure.com/msazure/';
 $project = 'One';
 $pipelineName = 'BicepMirror-Official'
 
+if($TagName -notmatch 'v\d+\.\d+\.\d+')
```

Comment:
nice sanity check :)

Reply:
I figured that if I'm going to cut off the `v` to produce a build number, I might as well validate it first.